pandas/tests/test_frame.py

```diff
@@ -7328,6 +7328,52 @@ def test_to_csv_path_is_none(self):
         recons = pd.read_csv(StringIO(csv_str), index_col=0)
         assert_frame_equal(self.frame, recons)
 
+    def test_to_csv_compression_gzip(self):
+        ## GH7615
```

Comment:
can you add a test to ensure that the ``ValueError`` is raised when passing an invalid string as compression